module-system/sov-modules-api/src/capabilities.rs

```diff
@@ -0,0 +1,62 @@
+#![deny(missing_docs)]
+
+//! The rollup capabilities module defines "capabilities" that rollup must
+//! provide if they wish to use the standard app template.
+//! If you don't want to provide these capabilities,
+//! you can bypass the Sovereign module-system completely
+//! and write a state transition function from scratch.
+//! [See here for docs](https://github.com/Sovereign-Labs/sovereign-sdk/blob/nightly/examples/demo-stf/README.md)
+
+use sov_rollup_interface::da::BlobReaderTrait;
+use sov_state::WorkingSet;
+
+use crate::{Context, Spec};
+
+/// Container type for mixing borrowed and owned blobs.
+pub enum BlobRefOrOwned<'a, B: BlobReaderTrait> {
```

Comment:
Is it possible to use https://doc.rust-lang.org/std/borrow/enum.Cow.html instead?